### PR TITLE
PAAS-1982: add package to fix jsessionid on jahia 8.1.1.0

### PR DESCRIPTION
**Pull request by Ronan-Gorain**
JIRA issue: https://jira.jahia.org/browse/PAAS-1982

Short description:

### CODE REVIEW

File: packages/one-shot/paas-1982-fix-jessionid.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - if ( "${globals.jahiaVersion}" != "8.1.1.0"):
+        - return:
+            type: success
+            message: "Jahia version is ${globals.jahiaVersion}, not 8.1.1.0 ; aborting"
+
+  fixWebXML:
+    - cmd[${this}]: |-
+        files=(/opt/tomcat/webapps/ROOT/WEB-INF/web.xml /constant_data/jahia/tomcat/webapps/ROOT/WEB-INF/web.xml)
+        for file in "${files[@]}"; do
+          line=$(grep -Fn sessionCreate  $file | awk '{print substr($1,0, length($1)-1)}')
```
Comment (laurentfufu) [inline packages/one-shot/paas-1982-fix-jessionid.yml:29]: same as https://github.com/Jahia/jahiastic-jahia/pull/182#discussion_r829807833

File: packages/one-shot/paas-1982-fix-jessionid.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          if [ ! -z $line ]; then
+            start=$((line-1))
+            end=$((line+2))
+
+            before=$(md5sum $file)
+            sed -i -e "$start s/^ /<\!-- /" -e "$end s/param>$/param>-->/"  $file
```
Comment (laurentfufu) [inline packages/one-shot/paas-1982-fix-jessionid.yml:35]: same as https://github.com/Jahia/jahiastic-jahia/pull/182#discussion_r829805816

File: packages/one-shot/paas-1982-fix-jessionid.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+actions:
+  checkJahiaVersion:
+    - getJahiaVersion
+    - if ( "${globals.jahiaVersion}" != "8.1.1.0"):
+        - return:
+            type: success
+            message: "Jahia version is ${globals.jahiaVersion}, not 8.1.1.0 ; aborting"
+
+  fixWebXML:
```
Comment (laurentfufu) [inline packages/one-shot/paas-1982-fix-jessionid.yml:25]: more a temporary workaround than a fix for us

**Review by laurentfufu — CHANGES_REQUESTED** [review]

File: packages/one-shot/paas-1982-fix-jessionid.yml (hunk 2 above)
Comment (Ronan-Gorain) [reply packages/one-shot/paas-1982-fix-jessionid.yml]: updated

File: packages/one-shot/paas-1982-fix-jessionid.yml (hunk 3 above)
Comment (Ronan-Gorain) [reply packages/one-shot/paas-1982-fix-jessionid.yml]: updated

File: packages/one-shot/paas-1982-fix-jessionid.yml (hunk 1 above)
Comment (Ronan-Gorain) [reply packages/one-shot/paas-1982-fix-jessionid.yml]: updated

**Review by laurentfufu — APPROVED** [review]

**Pull request merged**